ci/prod/cloudbuild.yaml

```diff
@@ -34,6 +34,7 @@ steps:
         --target build \
         -t us-central1-docker.pkg.dev/$PROJECT_ID/geyser-docker-repo/${_SERVICE_NAME}:build \
         --build-arg VITE_APP_API_ENDPOINT=${_API_ENDPOINT} \
+        --build-arg VITE_APP_BOLTZ_SWAP_DOMAIN=$$BOLTZ_SWAP_DOMAIN \
```

Comment:
The Botlz swap domain doesn't need to be a secret, it can just be an env var defined in the Build Trigger which you pass here with `_BOTLZ_SWAP_DOMAIN`